nats/src/main/java/io/micronaut/nats/reactive/PublishState.java

```diff
@@ -17,15 +17,12 @@
 
 import io.micronaut.core.annotation.Nullable;
 
-import javax.annotation.concurrent.Immutable;
-
 /**
  * Stores the state of a Nats message to be published.
  *
  * @author James Kleeh
  * @since 1.1.0
  */
-@Immutable
```

Comment:
Please add a comment to the javadocs saying something like "This class should be treated as immutable"

Reply:
I've added a comment